lti_emailer/canvas_api_client.py

```diff
@@ -21,6 +21,21 @@
 SDK_CONTEXT = SessionInactivityExpirationRC(**settings.CANVAS_SDK_SETTINGS)
 
 
+def get_section(canvas_course_id, section_id):
```

Comment:
It is highly likely that the sections by course id cache will still be valid when this page gets hit and if not, no worries, that means the other page will load that much faster after people visit this page. I think the point here is that we're caching the same information twice unnecessarily. It would be good to avoid that.